client/src/pages/layout/Navbar.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import {
+    AppBar,
+    Toolbar,
+    Typography,
+    Button,
+    Avatar,
+    List,
+    ListItem,
+    Badge,
+    Menu,
+    MenuItem,
+    Link,
+} from '@material-ui/core';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+// import { Link as RouterLink} from 'react-router-dom';
+
+import logo from '../../images/logo_study.png';
+import profileImg from '../../images/profile-pic.png';
+import { useStyles } from './NavbarStyles';
+import { useGlobalContext } from '../../context/studyappContext';
+
+const Navbar = () => {
+    const classes = useStyles();
+    const { isLoading } = useGlobalContext();
+
+    const [anchorEl, setAnchorEl] = useState(null);
+    const open = Boolean(anchorEl);
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    console.log('test context', isLoading);
+
+    return (
+        <div className={classes.root}>
+            <AppBar position="static" color="default" elevation={2}>
+                <Toolbar style={{ height: '100px', backgroundColor: '#FFF' }}>
```

Comment:
Try to avoid inline styling wherever you can: I know it's appealing when building the layout but can get easily lost in the code when also combined with useStyles.

Reply:
I have removed the inline styles.